doc/help/kdb.md

```diff
@@ -1,5 +1,4 @@
-kdb(1) -- key database access tools
-===================================
+# kdb(1) -- key database access tools
```

Comment:
Does this work with ronn?

Reply:
Yep. At least the man page for `kdb` still looks the same on my system.